test/test_ops.py

```diff
@@ -539,6 +539,19 @@ def test_errors(self, device, op):
                 out = op(si.input, *si.args, **si.kwargs)
                 self.assertFalse(isinstance(out, type(NotImplemented)))
 
+    @skipMeta
+    @onlyNativeDeviceTypes
+    @ops([op for op in op_db if op.error_inputs_sparse_func is not None], dtypes=OpDTypes.none)
+    @parametrize("layout", (torch.sparse_csr, torch.sparse_csc, torch.sparse_bsr, torch.sparse_bsc, torch.sparse_coo))
+    def test_errors_sparse(self, device, op, layout):
+        if not op.supports_sparse_layout(layout):
+            self.skipTest('unsupported layout')
```

Comment:
> Do we need this? If we explicitly put something in the error inputs function, shouldn't that automatically imply that the layout is supported in general?

We have one `error_inputs_sparse_func` for all five sparse layouts. So, when running error inputs tests, it would be better to see that the test_errors_sparse test is skipped for an unsupported layout rather than seeing the test being (false-positive) successful because no error inputs are generated.

That said, I'll move this skipTest into `error_inputs_sparse` function so that the test writers will not need to worry about skipping tests for unsupported layouts.